newrelic-infra-operator.yaml

```diff
@@ -19,7 +19,7 @@ pipeline:
     with:
       repository: https://github.com/newrelic/newrelic-infra-operator
       tag: v${{package.version}}
-      expected-commit: bbdc9a67325c38e5a414a6627f8833ee26a9febc
+      expected-commit: 0e9674e331e7acf2eca697bd509058e23e13cb4f
```

Comment:
what we do here, looks like this changed the hash
cc @rawlingsj

Reply:
yeah just looking now - initial thought was upstream recreated the tag with a new commit but now I'm not so sure, might be related to the update config below.  Investigating.